app/components/works/available_date_component.html.erb

```diff
@@ -1,40 +1,43 @@
-<div data-controller="complex-radio available-date">
-  <div class="mb-3 row">
-    <div class="col-sm-10 release-option" data-complex-radio-target="selection">
-      <%= form.radio_button :release, 'immediate', class: 'form-check-input',
-                            data: {
-                              action: 'complex-radio#disableUnselectedInputs change->auto-citation#updateDisplay available-date#clearError'
-                            } %>
-      <%= form.label :release_immediate, 'Immediately once deposit is completed' %>
-    </div>
-  </div>
-
-  <div class="embargo-date mb-5">
-    <div class="row<%= ' is-invalid' if error? %>">
+<fieldset>
+  <legend class="h5">Release date</legend>
```

Comment:
Is this supposed to be hidden? I don't see it in the mockups.  Can you add screenshots that shows the diff?

Reply:
indeed...I just hid both legends using a `visually-hidden` class and otherwise left the displays as before...so it now looks exactly the same as it did before (just verified against main branch on localhost)